webapp/tests/forms/steps/lotse/test_pauschbetrag.py

```diff
@@ -1,37 +1,52 @@
+import pytest
+
 from app.forms.steps.lotse.pauschbetrag import calculate_pauschbetrag
 
 
 class TestCalculatePauschbetrag:
 
-    def test_if_no_merkzeichen_or_pflegegrad_set_then_return_correct_value_for_beh_grad(self):
+    def test_if_no_merkzeichen_or_pflegegrad_set_then_return_correct_value_for_disability_degree(self):
         input_output_pairs = [
             (20, 384),
+            (25, 384),
             (30, 620),
+            (35, 620),
             (40, 860),
+            (45, 860),
             (50, 1140),
+            (55, 1140),
             (60, 1440),
+            (65, 1440),
             (70, 1780),
+            (75, 1780),
             (80, 2120),
+            (85, 2120),
             (90, 2460),
+            (95, 2460),
```

Comment:
This on the other hand is a great catch that we definitely need to test and make sure works correctly.